index.d.ts

```diff
@@ -149,6 +149,13 @@ declare namespace captureWebsite {
 		*/
 		readonly disableAnimations?: boolean;
 
+		/**
+		Options to enable javascript execution on the page. If disabled, it also disable modules and scripts
```

Comment:
This could be better written. Spend some more time on the description. This and the readme text should be in sync.